core/domain/feedback_services.py

```diff
@@ -115,8 +118,16 @@ def create_message(
             thread.subject = updated_subject
     thread.put()
 
+    if (user_services.is_user_registered(author_id) and len(text) > 0 and
```

Comment:
Yup, I will file this bug. I do not think it as a bug may be a new feature. Since original #461 mentions only sending feedback messages and nothing about suggestions. I think we can use the current infrastructure for suggestion, for e.g. same taskqueue can be used for this purpose but handler function will be different for suggestions, since it has to be processed differently.